pkg/session/bootstrap.go

```diff
@@ -1369,6 +1372,77 @@ var (
 	SupportUpgradeHTTPOpVer int64 = version174
 )
 
+func acquireLock(s sessiontypes.Session) bool {
```

Comment:
you have define returen of  acqurieLock, but caller did not handle the acquireLock fail case.